### DISCUSSION TITLE
enableDragSelection on update

### DISCUSSION THREAD

**Discussion by vitto32** [katspaugh/wavesurfer.js#2967] [Q&A]
Is it possible to add a listener to "update" events triggered by newly created regions (via the enableDragSelection method)?

I'm using enableDragSelection in combination with autoScroll and same wired stuff happens if the selection starts while the wave is scrolling (audio is playing). I'd like to do something when the user starts selection, eg. pause the audio or disable autoscroll and then restore it when the selection ends (I can intercept this event via 'region-created')

**Comment by katspaugh** [comment]
Feel free to add an event like `drag-start`, there's currently no event emitted.

**Comment by Schwankenson** [comment]
Much better workaround:

```
regionPlugin.on("region-initialized", async (region: Region) => {
      const currentRegion = region;

      region.on("update", (side) => {
        console.log("UPDATE", side, currentRegion.start, currentRegion.end);        
        const content = document.createElement("div");
        content.textContent = `${Math.round(
          currentRegion.end - currentRegion.start
        )}s`;
        region.setContent(content as HTMLElement);
      });
    }
```
